### ISSUE TITLE
Multiple symbols shown for one POI

### ISSUE THREAD

**Issue by masc4ii** [treee111/wahooMapsCreator#159]
## Expected Behavior
For a POI, one symbol should be displayed on the screen.

## Current Behavior
For some POIs, the symbol is displayed multiple times. This happens e.g. for POIs on larger buildings, and when zooming in. For POIs with the tag "type=multipolygon" the symbol will move on the entire screen like a ghost, when moving with your bike (e.g. castle of Schmalkalden, tile 135/86).

## Steps to Reproduce the Issue
1. Search for a POI on a large building. Maybe use cruiser application - that is easier and shows the bug too. If you create the map including historic-castle, the castle of Schmalkalden will show the bug, same as many many castles in Italy Toskana. But also many café's in "way"-items in nearly all cities will show the bug.
2. zoom in -> multiple same symbols are displayed for one POI
3. for castle of Schmalkalden: move the map and the "ghost castle" will be displayed.

## Context
I think, the bug will happen for "add-more-pois" branch. Not sure for your main branch.

## Solution
In wahoomc/resources/tag_wahoo_adjusted)/tag-wahoo-poi.xml, add for all POI lines `label-position="true"`
It should look like this:
```
  <!-- ************* POIS *************** --> 
  <pois>
        <osm-tag key="amenity" label-position="true" value="fuel" zoom-appear="14" /> 
        <osm-tag key="amenity" label-position="true" value="cafe" zoom-appear="14" /> 
...
  </pois>
  <ways>
        <osm-tag key="amenity" label-position="true" value="fuel" zoom-appear="14" /> 
        <osm-tag key="amenity" label-position="true" value="cafe" zoom-appear="14" /> 
...
```
Now the symbol will be displayed just once per, and exactly in the middle of the POI item.
(Maybe it is just needed for "ways" and not for "pois" - to be tested.)

**Comment by masc4ii** [comment]
I did more testing:
what I wrote above is needed for ways (often polygons) only, and not for pois (is a point already).

Before:
<img width="687" alt="Bildschirmfoto 2022-10-27 um 20 23 44" src="https://user-images.githubusercontent.com/30245296/198369510-d8242744-665e-4e8a-b87a-43c668719332.png">

After:
<img width="687" alt="Bildschirmfoto 2022-10-27 um 20 24 06" src="https://user-images.githubusercontent.com/30245296/198369490-cf82c27c-b1dd-4774-88f6-1445e3855ba3.png">


Before: (castle in the middle of the picture is moving, when moving the map)
<img width="687" alt="Bildschirmfoto 2022-10-27 um 20 28 06" src="https://user-images.githubusercontent.com/30245296/198370057-29626d4b-3f20-473a-925d-4ef892154bfb.png">

After:
<img width="687" alt="Bildschirmfoto 2022-10-27 um 20 28 23" src="https://user-images.githubusercontent.com/30245296/198370064-f8e93ef0-78b9-48ef-ac8d-3ee449f9b905.png">

**Comment by treee111** [comment]
Hi @masc4ii,

nice detailled finding and input for making it better!
Seams that it comes with the ways-tags.

I had a little hustle to find the position of your example, but with the coordinates I finally found it.
Reproduction was relatively straight forward.

I made this 3 examples confirming your suggested solution:
1. branch `develop`, zoom 50m in cruiser:
![grafik](https://user-images.githubusercontent.com/53038537/198899708-05ef0ce1-a862-470b-8853-8a10ab64ad38.png)

2. branch `add-more-pois` zoom 50m in cruiser:
![grafik](https://user-images.githubusercontent.com/53038537/198899730-d22350e6-8def-4086-8fa3-906600a72465.png)

3. branch `add-more-pois` zoom 50m in cruiser and `label-position="true"` for cafe way-tags:
![grafik](https://user-images.githubusercontent.com/53038537/198899891-44c9bc63-8ce9-4a6a-bd95-84c3e19bae09.png)

Given this series, I thinkg adding  `label-position="true"` for way-tags makes sense!

Yesterday morning I had a similar experience with this cemetery:
https://openstreetbrowser.org/#map=17/48.89035/9.39906&categories=public%5Btype%3Damenity%253Ddrinking_water%5D

I have the same output when using
- branch `develop`
- branch `add-more-pois` and added `label-position="true"` for drinking_water way- and pois-tags
![grafik](https://user-images.githubusercontent.com/53038537/198900383-6480e9b2-97a3-4a1d-9e21-67a6df3b2de3.png)

`label-position="true"` doesn't help here and seams not to be similar to your findings.

Is this another story and OK that there are so many symbols?

**Comment by masc4ii** [comment]
It looks like there are so many symbols already in OSM data. Here I have no solution yet. 
Same for this castle here - I think one symbol would be enough - but there are 4 tags added, so 4 symbols shown: https://www.openstreetmap.org/way/58883585
<img width="740" alt="Bildschirmfoto 2022-10-30 um 21 49 56" src="https://user-images.githubusercontent.com/30245296/198900900-d7d03aab-e9c8-43eb-a383-9750f635fb03.png">

**Comment by treee111** [comment]
that were also my thoughts about that. So I'll add your suggestion to the ways-tags :-)

**Comment by treee111** [comment]
fixed via #158
